backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/graphql/GraphqlCustomTypes.java

```diff
@@ -65,4 +70,37 @@ public BinaryFileWrapper parseLiteral(Object input) {
                 }
               })
           .build();
+
+  public static final GraphQLScalarType GraphQLDuration =
+      GraphQLScalarType.newScalar()
+          .name("Duration")
+          .description("A ISO 8601 duration scalar with year, month, week and day components.")
+          .coercing(
+              new Coercing<Integer, String>() {
+                @Override
+                public String serialize(Object input) throws CoercingSerializeException {
+                  return TypeUtils.intToDuration(input);
+                }
+
+                @Override
+                public Integer parseValue(Object input) throws CoercingParseValueException {
```

Comment:
Normally we do this in the backend. So that csv imports would also be parsed. 

I am missing test cases to help me understand what this is supposed to do.